lk2nd/device/dts/msm8952/msm8976-qrd.dts

```diff
@@ -9,6 +9,67 @@
 };
 
 &lk2nd {
+	bq-x5-plus {
+		model = "BQ X5 Plus (Longcheer L9360)";
+		compatible = "longcheer,l9360";
+		lk2nd,match-panel;
+
+		lk2nd,dtb-files = "msm8976-longcheer-l9360";
+
+		panel {
+			compatible = "longcheer,l9360-panel", "lk2nd,panel";
+
+			qcom,mdss_dsi_adv7533_1080p {
+				compatible = "longcheer,adv7533-1080p";
+			};
+			qcom,mdss_dsi_adv7533_720p {
+				compatible = "longcheer,adv7533-720p";
+			};
+			qcom,mdss_dsi_booyi_hx8399b_1080p_command {
+				compatible = "longcheer,booyi-hx8399b";
+			};
+			qcom,mdss_dsi_booyi_hx8399c_1080p_video {
+				compatible = "longcheer,booyi-hx8399c";
+			};
+			qcom,mdss_dsi_hx8399a_1080p_video {
+				compatible = "longcheer,hx8399a-video";
+			};
+			qcom,mdss_dsi_nt35523_wqxga_video {
+				compatible = "longcheer,nt35523-video";
+			};
+			qcom,mdss_dsi_nt35597_dsc_wqxga_cmd {
+				compatible = "longcheer,nt35597-dsc-cmd";
+			};
+			qcom,mdss_dsi_nt35597_dsc_wqxga_video {
+				compatible = "longcheer,nt35597-dsc-video";
+			};
+			qcom,mdss_dsi_nt35597_wqxga_cmd {
+				compatible = "longcheer,nt35597-cmd";
+			};
+			qcom,mdss_dsi_nt35597_wqxga_video {
+				compatible = "longcheer,nt35597-video";
+			};
+			qcom,mdss_dsi_otm1906c_1080p_cmd {
+				compatible = "longcheer,otm1906c-1080p-cmd";
+			};
+			qcom,mdss_dsi_truly_1080p_cmd {
+				compatible = "longcheer,truly-1080p-cmd";
+			};
+			qcom,mdss_dsi_truly_1080p_video {
+				compatible = "longcheer,truly-1080p-video";
+			};
+			qcom,mdss_dsi_truly_nt35695_1080p_command {
+				compatible = "longcheer,truly-nt35695";
+			};
+			qcom,mdss_dsi_truly_otm1906a_1080p_command {
+				compatible = "longcheer,truly-otm1906a";
+			};
+			qcom,mdss_dsi_truly_r63350_1080p_video {
+				compatible = "longcheer,truly-r63350";
+			};
+		};
+	};
```

Comment:
```suggestion
		panel {
			compatible = "longcheer,l9360-panel", "lk2nd,panel";

			qcom,mdss_dsi_booyi_hx8399b_1080p_command {
				compatible = "longcheer,booyi-hx8399b";
			};
			qcom,mdss_dsi_booyi_hx8399c_1080p_video {
				compatible = "longcheer,booyi-hx8399c";
			};
			qcom,mdss_dsi_truly_nt35695_1080p_command {
				compatible = "longcheer,truly-nt35695";
			};
			qcom,mdss_dsi_truly_otm1906a_1080p_command {
				compatible = "longcheer,truly-otm1906a";
			};
			qcom,mdss_dsi_truly_r63350_1080p_video {
				compatible = "longcheer,truly-r63350";
			};
		};
	};
```

These look like the default qrd ones from i.e. https://android.googlesource.com/kernel/msm.git/+/android-msm-lego-3.10-marshmallow-dr/arch/arm/boot/dts/qcom/msm8976-mdss-panels.dtsi

but would be nice if you double-check. If you have decompiled dtb, can you check if some of them have different style of setting `...-panel-name` string? I think sometimes it's obvious when N last ones are different and added by the OEM instead of being from the qcom design...

I don't exactly like the first 4 left in there either so would be funny if it's actually only the last one added by the OEM...

Reply:
I removed the entries that match the default ones.